src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -354,7 +354,7 @@ public abstract partial class Array : System.Collections.ICollection, System.Col
         public static int LastIndexOf<T>(T[] array, T value) { throw null; }
         public static int LastIndexOf<T>(T[] array, T value, int startIndex) { throw null; }
         public static int LastIndexOf<T>(T[] array, T value, int startIndex, int count) { throw null; }
-        public static void Resize<T>([System.Diagnostics.CodeAnalysis.NotNullAttribute] ref T[]? array, int newSize) { }
+        public static void Resize<T>([System.Diagnostics.CodeAnalysis.NotNullAttribute] ref T[]? array, int newSize) { throw null; }
```

Comment:
Interesting consequence of the recent `[NotNull]` change is that the compiler now treats it as an _out_-like parameter: you must assign a non-null value to it before method exit. So this means our ref assemblies' dummy method bodies need to throw, just as they would for any other method which has an _out_ parameter.

I didn't see any other cases where this needed to be changed, but might have implications for our "generate ref asms automatically" tool.